src/main/java/org/folio/clients/ClientUtil.java

```diff
@@ -88,7 +87,7 @@ public static JsonObject getResponseEntity(CloseableHttpResponse response) {
         LOGGER.error("Exception while building response entity", e);
       }
     }
-    return null;
+    throw new IOException("Get invalid response with status: " + response.getStatusLine().getStatusCode());
```

Comment:
I'm guessing we need to be more clear with what the exception is. We can go ahead and merge it for now, but while testing if we see any issues, let's come back to improve this